R/new-dataset.R

```diff
@@ -65,7 +65,7 @@ newDatasetFromFile <- function (file, name=basename(file), ...) {
         halt("File not found")
     }
     ds <- createDataset(name=name, ...)
-    ds <- addSourceToDataset(ds, createSource(file))
+    ds <- addSourceToDataset(ds, createSource(file), savepoint=FALSE)
```

Comment:
`autorollback` will be the only option that users will provide usually, `savepoint=false` is a way to disable not only the rollback but the savepoint at all. Usually, like for merge the savepoint is left behind even if the user decided not to rollback to it, so it can diagnose the issue and then decided to rollback later.

`savepoint=false` totally disables the savepoint. This is done by functions that create new datasets, as there is nothing we should rollback to (previously the dataset didn't exist) and some clients might decide to delete the dataset if creation fails (whaam does so) a rollback would be pointless and inefficient. 

`savepoint=true` is actually the default so everything that appends data is working with `savepoint=true`, only the `newDataset*` functions provide `savepoint=false` and I expect we don't expose that option to users is they rely on the `new/append` functions